examples/spellchecker/src/main.rs

```diff
@@ -0,0 +1,148 @@
+use bindings::windows::win32;
+use win32::com;
+use win32::intl;
+use windows::*;
+
+fn main() {
+    let input = std::env::args()
+        .nth(1)
+        .expect("Expected one command line argument for text to be spell-corrected");
+    unsafe {
+        // Initialize COM apartment
+        com::CoInitializeEx(
+            std::ptr::null_mut(),
+            com::COINIT::COINIT_MULTITHREADED.0 as u32,
+        )
+    }
+    .unwrap();
+
+    // Create ISpellCheckerFactory
+    let factory = create_factory();
+
+    // Make sure that the "en-US" locale is supported
+    let mut supported: BOOL = false.into();
+    let locale: Vec<u16> = "en-US\0".encode_utf16().collect();
+    // https://github.com/microsoft/windows-rs/issues/489
+    factory
+        .IsSupported(locale.as_ptr(), &mut supported as *mut _ as *mut i32)
```

Comment:
Just FYI - I plan to map trailing out params as `Result<T>` so that this would simply become:

```rust
let supported = factory.IsSupported(...).ok()?;
```

Reply:
Is there any issue tracking this? I think it would be great to do this as it would make developing around these interfaces a lot easier.

Reply:
https://github.com/microsoft/windows-rs/issues/509